packages/fa-icon-chooser/dev/runtime.js

```diff
@@ -72,7 +72,11 @@ const FaIconChooserDevExports = (function () {
     document.querySelectorAll('#result pre').forEach(child => child.remove());
   }
 
+  const svgEmbedOverrideCallback = (window['__FA_SVG_EMBED__'] = () => true);
+
   function addIconChooser(props) {
+    svgEmbedOverrideCallback();
```

Comment:
We don't need to invoke the callback. That's going to do is return `true` here in this function, which is useless.

```suggestion

```